src/commands/misc/helpRemember.js

```diff
@@ -0,0 +1,37 @@
+const { EmbedBuilder } = require('@discordjs/builders')
+
+module.exports = {
+    name: 'help-remember',
```

Comment:
If we are planning to add subcommands later, would it be better for this to be a subcommand itself rather than its own command?

Reply:
Most likely, yes